docs/observability/recipes-observability.md

```diff
@@ -8,6 +8,14 @@
 
 [Github Repo](https://github.com/MagdaPaj/application-insights-aspnet-sample-trace-context-propagation).
 
+## Example: OpenTelemetry over a message oriented architecture in Java with Jaeger, Prometheus and Azure Monitor
+
+[Github Repo](https://github.com/iamnicoj/OpenTelemetry-Async-Java-with-Jaeger-Prometheus-AzMonitor)
```

Comment:
@iamnicoj - This repo is a great example of self contained resource for upskilling in Open Telemetry. I would like to see if you would be okay for this to be pushed out as a self serve upskilling material

Reply:
Of course, this can we pushed out as a self serve upskilling material!